src/app/app.html

```diff
@@ -26,6 +26,6 @@
     </ion-list>
   </ion-content>
 </ion-menu>
-<ion-nav id="myNav" #myNav [root]="rootPage"></ion-nav>
+<ion-nav id="myNav" #myNav [root]="rootPage" ></ion-nav>
```

Comment:
Fat fingered? Don't think this space needs to be here

Reply:
fat fingered confirmed